styleguide/markdown-syntax-style.md

```diff
@@ -1,157 +1,88 @@
-If something isn't included in this guide, see the [Microsoft Style Guide](https://docs.microsoft.com/en-us/style-guide/welcome/).
+Welcome to the Buildkite Markdown syntax style guide.
 
-Table of contents:
-* [Language](#Language)
-* [Style and formatting](#Style-and-formatting)
-* [Code and filenames](#Code-and-filenames)
-* [Working with the docs site](#Working-with-the-docs-site)
-* [Screenshots](#Screenshots)
-* [GraphQL API schema](#graphql-api-schema)
-
-## Language
-
-This section covers everything related to the language and formatting used in Buildkite documentation (and website, to some extent).
-
-### Dictionary
-
-We use American English, so our first-call dictionary is [Merriam Webster](https://www.merriam-webster.com/). It is not a single source of truth, just a good starting/reference point.
-We also use [Vale](https://github.com/apps/vale-linter), our own [linter](https://github.com/buildkite/docs#linting) with a file that contains our own [custom list of words](https://github.com/buildkite/docs/blob/main/vale/vocab.txt) that may confuse the spellchecker, [filename linter](https://ls-lint.org/1.x/getting-started/introduction.html), and a [markdown linter](https://github.com/DavidAnson/markdownlint).
-
-### English flavor
-Again, we use American English. We're also a highly multi-national team, so here is the list of the most notable [differences between American, British, and Australian English](http://web.archive.org/web/20180925084522/http://www.linktranslation.com.au/the-differences-between-american-british-and-australian-english/) English to watch out for.
-
-### Commas
-
-Keep it clean and uncluttered. If the sentence can live without that comma - leave it out.
-
-### Serial commas
-
-Absolutely yes, yes, and yes.
-
-Serial commas are also known as 'Oxford commas'.
+These guidelines provide details about the specific Markdown syntax used to write the Buildkite docs, as well as its file structure and how to work with the site (useful to understand when adding new pages), and screenshots.
 
-### Active/passive voice
+For details about the Markdown syntax used to render your writing, refer to the [Writing style guide](writing-style.md).
 
-Generally, use active voice. It's usually clearer and simpler.
-
-### Gender
-
-Always use ‘they’, never use ‘he’ or ‘she’.
-More info on writing about pronouns and in the [Microsoft Style Guide](https://docs.microsoft.com/en-us/style-guide/grammar/nouns-pronouns).
-
-### Talking about YAML
-
-YAML looks more simple than it is.
-It takes some care and discipline to write about.
-See [Talking about YAML](./yaml.md) for complete guidance.
+Table of contents (main headings):
+* [Style and formatting](#style-and-formatting)
+* [Code and filenames](#code-and-filenames)
+* [Working with the docs site](#working-with-the-docs-site)
+* [Screenshots](#screenshots)
+<!---
+* [GraphQL API schema](#graphql-api-schema)
```

Comment:
This appears to have been moved somewhere else...

Reply:
Should this link here?
https://github.com/buildkite/docs/blob/style-guide-updates-language-and-markdown-syntax/data/graphql/schema.graphql

Reply:
As discussed, we'll leave a link to this schema in the README.

For some reason, the README also has a link on how to update the GraphQL docs (as part of its 'Updating the navigation') section, although I'm not sure if this is still a thing?
The 'Working with the docs site' section (now in the Markdown syntax style guide) has nothing on GraphQL.